src/telegramServer.ts

```diff
@@ -217,29 +217,37 @@ export class TelegramServer extends EventEmitter {
     }
   }
 
-  /**
-   * @FIXME
-   * (node:103570) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11
-   * EditedMessageText listeners added to [TelegramServer]. Use emitter.setMaxListeners() to
-   * increase limit (Use `node --trace-warnings ...` to show where the warning was created)
-   */
   async waitBotEdits() {
     return new Promise<void>((resolve) => {
-      this.on('EditedMessageText', () => resolve());
+      const handler = () => {
+        this.off('EditedMessageText', handler);
+        resolve();
+      };
+      this.on('EditedMessageText', handler);
```

Comment:
I suppose we can simple use `once` onstead of `on` - the we don't have to remove listener manually.